### ISSUE TITLE
No main class?

### ISSUE THREAD

**Issue by elmirandae** [alvarag/LSH-IS#1]
Hi!
I'm trying to add this filter to Weka, but I've got a mistake from GIT's bash at the last step when I introduce the next command:
java -cp instanceselection.jar:weka.jar weka.gui.GUIChooser

The GIT's bash prompt says: 
Error: could not find or load main class weka.gui.GUIChooser

I've got already installed and configured Apache Ant 1.9.7 and Java

**Comment by alvarag** [comment]
Hi,

The ```weka.gui.GUIChooser``` is the main class of the Weka GUI. The error seems to be that weka.jar can not be found. ¿Are you launching the application from ```dist/weka``` directory?

Regards,

**Comment by elmirandae** [comment]
Hi! Thanks for answering. I have been trying but is not possible yet to do it... am I doing something wrong? 
- I've got installed and configured Apache Ant, I'm executing the command into LSH-IS/dist/weka directory. I've got installed Weka 3.8

**Comment by alvarag** [comment]
Hi, maybe the problem was that the README file had a mistake. The build command with ant is ```dist_weka``` and not ```dist_all```.

Here you can see the whole process:
```shell
~/tmp $ git clone https://github.com/alvarag/LSH-IS.git
Clonar en «LSH-IS»...
remote: Counting objects: 137, done.
remote: Compressing objects: 100% (3/3), done.
remote: Total 137 (delta 0), reused 0 (delta 0), pack-reused 134
Receiving objects: 100% (137/137), 6.13 MiB | 1.05 MiB/s, done.
Resolving deltas: 100% (18/18), done.
Comprobando la conectividad… hecho.
~/tmp $ cd LSH-IS/
~/tmp/LSH-IS $ ant dist_weka
Buildfile: /home/alvar/tmp/LSH-IS/build.xml

init_all:

init_compile:
    [mkdir] Created dir: /home/alvar/tmp/LSH-IS/build/is

compile:
    [javac] /home/alvar/tmp/LSH-IS/build.xml:49: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds
    [javac] Compiling 1 source file to /home/alvar/tmp/LSH-IS/build/is
    [javac] /home/alvar/tmp/LSH-IS/build.xml:54: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds
    [javac] Compiling 4 source files to /home/alvar/tmp/LSH-IS/build/is

init_dist:
    [mkdir] Created dir: /home/alvar/tmp/LSH-IS/dist
    [mkdir] Created dir: /home/alvar/tmp/LSH-IS/dist/lib
    [mkdir] Created dir: /home/alvar/tmp/LSH-IS/dist/weka

dist:
      [jar] Building jar: /home/alvar/tmp/LSH-IS/dist/lib/instanceselection.jar

dist_weka:
     [copy] Copying 1 file to /home/alvar/tmp/LSH-IS/dist/weka
     [copy] Copying 1 file to /home/alvar/tmp/LSH-IS/dist/weka

BUILD SUCCESSFUL
Total time: 1 second
~/tmp/LSH-IS $ cd dist/weka/
~/tmp/LSH-IS/dist/weka $ java -cp instanceselection.jar:weka.jar weka.gui.GUIChooser 
```

**Comment by elmirandae** [comment]
Hi! Thanks again for answer. I tried with the instrutions you left above. This is what I've got.

miran@HP-Red360 MINGW64 ~
$ git clone https://github.com/alvarag/LSH-IS.git
Cloning into 'LSH-IS'...
remote: Counting objects: 137, done.
remote: Total 137 (delta 0), reused 0 (delta 0), pack-reused 137
Receiving objects: 100% (137/137), 6.13 MiB | 595.00 KiB/s, done.
Resolving deltas: 100% (19/19), done.

miran@HP-Red360 MINGW64 ~
$ cd LSH-IS

miran@HP-Red360 MINGW64 ~/LSH-IS (master)
$ ant dist_weka
Buildfile: C:\Users\miran\LSH-IS\build.xml

init_all:

init_compile:
    [mkdir] Created dir: C:\Users\miran\LSH-IS\build\is

compile:
    [javac] C:\Users\miran\LSH-IS\build.xml:49: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds
    [javac] Compiling 1 source file to C:\Users\miran\LSH-IS\build\is
    [javac] C:\Users\miran\LSH-IS\build.xml:54: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds
    [javac] Compiling 4 source files to C:\Users\miran\LSH-IS\build\is

init_dist:
    [mkdir] Created dir: C:\Users\miran\LSH-IS\dist
    [mkdir] Created dir: C:\Users\miran\LSH-IS\dist\lib
    [mkdir] Created dir: C:\Users\miran\LSH-IS\dist\weka

dist:
      [jar] Building jar: C:\Users\miran\LSH-IS\dist\lib\instanceselection.jar

dist_weka:
     [copy] Copying 1 file to C:\Users\miran\LSH-IS\dist\weka
     [copy] Copying 1 file to C:\Users\miran\LSH-IS\dist\weka

BUILD SUCCESSFUL
Total time: 11 seconds

miran@HP-Red360 MINGW64 ~/LSH-IS (master)
$ cd dist/weka

miran@HP-Red360 MINGW64 ~/LSH-IS/dist/weka (master)
$ java -cp instanceselection.jar:weka.jar weka.gui.GUIChooser
Error: no se ha encontrado o cargado la clase principal weka.gui.GUIChooser


Hope you can help me, I don't know what could be worng. I'm really interested in perform this algorithm it's for my thesis :(

**Comment by alvarag** [comment]
Do the .jar files exist in the folder ```dist/weka```?

**Comment by elmirandae** [comment]
Yes they does, there are two files into the path (instanceselection.jar 10 KB and weka.jar 7, 1 KB

**Comment by xiebb123456** [comment]
> Yes they does, there are two files into the path (instanceselection.jar 10 KB and weka.jar 7, 1 KB
using this commond:

java -cp instanceselection.jar;weka.jar weka.gui.GUIChooser
